src/megachatapi_impl.cpp

```diff
@@ -1885,6 +1885,51 @@ MegaChatListItemList *MegaChatApiImpl::getChatListItems()
     return items;
 }
 
+MegaChatListItemList *MegaChatApiImpl::getChatListItemsByPeers(megachat::MegaChatPeerList *peers)
```

Comment:
Here we need to remove `megachat` namespace in the parameters. Otherwise, cross-compilers such as Android NDK will complain